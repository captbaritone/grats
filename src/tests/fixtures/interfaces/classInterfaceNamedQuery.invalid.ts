/** @gqlInterface */
class Query {
  /** @gqlField */
  field: string;
}
