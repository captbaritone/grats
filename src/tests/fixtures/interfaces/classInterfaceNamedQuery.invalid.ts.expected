-----------------
INPUT
----------------- 
/** @gqlInterface */
class Query {
  /** @gqlField */
  field: string;
}

-----------------
OUTPUT
-----------------
src/tests/fixtures/interfaces/classInterfaceNamedQuery.invalid.ts:2:1 - error: Query root type must be Object type, it cannot be Query.

2 class Query {
  ~~~~~~~~~~~~~
3   /** @gqlField */
  ~~~~~~~~~~~~~~~~~~
4   field: string;
  ~~~~~~~~~~~~~~~~
5 }
  ~
